app/controllers/staff/speakers_controller.rb

```diff
@@ -70,7 +60,7 @@ def speaker_params
     params.require(:speaker).permit(:bio, :email, :user_id, :event_id, :speaker_name, :speaker_email)
   end
 
-  def set_proposal
-    @proposal = Proposal.find_by(uuid: params[:proposal_uuid])
+  def set_program_session
+    @program_session = ProgramSession.find_by(id: params[:program_session_id])
```

Comment:
This should be scoped by current_event.  We don't want to pull program sessions from another event, in case of id hacking.